src/review/review.service.ts

```diff
@@ -98,6 +99,34 @@ export class ReviewService {
     return reviews.map((rawReview) => this.transformReview(rawReview, null))
   }
 
+  async editReview(
```

Comment:
Since this method allows only `pending` review and `own` reviews. Pls name this method as `editMyPendingReview` (we may need `editMyReview` for editing their own approved reviews in the future)

Reply:
fixed